modules/AlphaWalletCore/AlphaWalletCore/WebImageURL.swift

```diff
@@ -84,17 +84,34 @@ public enum WebImageURL: Codable, Hashable, Equatable, CustomStringConvertible {
     }
 
     public init(url: URL, withUrlRewriting: Bool = true, rewriteGoogleContentSizeUrl size: GoogleContentSize = .s750) {
-        if let url = WebImageURL.functional.rewriteGoogleContentSizeUrl(url: url, size: size), withUrlRewriting {
+        let maybeIpfs = url.absoluteString
```

Comment:
Can you move this logic into in a propertly like with: `var rewriteIfIpfsOrNil`?